### PR TITLE
jcli - certificate get-vote-plan-id, implemented

### PR DESCRIPTION
**Pull request by rinor**
resolve https://github.com/input-output-hk/jormungandr/issues/2216

### CODE REVIEW

File: jcli/src/jcli_app/certificate/get_stake_pool_id.rs [hunk 1]
```diff
@@ -9,7 +9,7 @@ use structopt::StructOpt;
 pub struct GetStakePoolId {
     /// read the certificate from
     pub input: Option<PathBuf>,
-    /// write the certificate too
+    /// write the id to
```
Comment (NicolasDP) [inline jcli/src/jcli_app/certificate/get_stake_pool_id.rs:12]: while I appreciate the fix, it would be better split into a separate commit. Otherwise the git history will show that this was changed for adding the `get-vote-plan-id` while it is actually a wording change in the doc

**Comment by NicolasDP** [comment]
@rinor , if you rebase your PR the test should pass now

**Comment by rinor** [comment]
done
